stacks/README.md

```diff
@@ -3,3 +3,11 @@
 ### C or C++
 
 1. [Balanced Parenthesis](c-or-cpp/balanced-parenthesis.cpp)
+
+### Java 
```

Comment:
Please @sridhar-5 Rename the folder of java like this **java** not **JAVA** to make it like other directories.
like this [here](https://github.com/MakeContributions/DSA/tree/main/scheduling/java)

Reply:
I was not able to rename the folder for some reason can you help me here please. That is the only suggestion left if I manage to rename the folder. I think this is done.

Reply:
Done with the renaming as well.

Thanks.

Reply:
LGTM @sridhar-5 , Good work

Last thing please do the requested work below from @ming-tsai